jena-shex/src/main/java/org/apache/jena/shex/expressions/ShapeExprAtom.java

```diff
@@ -31,6 +31,7 @@ public class ShapeExprAtom extends ShapeExpression {
     private final ShapeExpression other;
 
     public ShapeExprAtom(ShapeExpression other) {
+        super(null);
```

Comment:
These super(null) are unnecessary. Add a protected constructor `ShapeExpression() { this(null); }`